components/comparision/ComparisonCard.vue

```diff
@@ -0,0 +1,63 @@
+<template>
```

Comment:
What is the difference between comparisonCard and comparisonOptionSelectCard?

Reply:
![Screenshot 2024-06-10 at 10 51 52 AM](https://github.com/formester/formester-site/assets/59838143/581625b9-997b-4a1c-9b83-08d5659500b0)
comparison card

![Screenshot 2024-06-10 at 10 52 16 AM](https://github.com/formester/formester-site/assets/59838143/c6d01fc3-9b76-4f41-9a8b-679fcf088a26)
Comparision option select card 
@ankitsinghaniyaz